### DISCUSSION TITLE
Statusreport-Woche 9

### DISCUSSION THREAD

**Discussion by elnaz-gharoon** [SE-TINF22B2/G3-ApeRepublic#95] [Statusreports]
ich möchte Sie daran erinnern, dass der letzte „Statusbericht“ in diesem Semester aus einer Statistik der Aufwände je Person und Phase des SE-Prozesses sowie der Mitarbeiterumfrage besteht.

**Personen und Aufwand:** 

1. Elnaz -> 15h + 4h Meetings
2. Anisa -> 6h + 4h Meetings
3. Marc  -> Projektmanagement-5h (4h Meetings), Konzept-11h, Design-0h, Total: 16h
4. Simon -> 6h + 4h Meetings
5. Matej -> 12h + 4h Meetings

**Comment by roland-KA** [comment]
Können diese Aufwände auch noch auf die Phasen des SE-Prozesses (RE, Design, ..., Projektmanagement) heruntergebrochen werden?
